lib/spack/spack/test/cmd/bootstrap.py

```diff
@@ -7,7 +7,7 @@
 import pytest
 
 import spack.config
-import spack.environment
+import spack.environment as ev
```

Comment:
Why this change?

Reply:
See comment in the main thread

Reply:
I think we can take care of understanding these two failures in a following PR.